src/Chunk.h

```diff
@@ -276,6 +276,12 @@ class cChunk :
 	bool DoWithEntityByID(UInt32 a_EntityID, cEntityCallback & a_Callback, bool & a_CallbackResult);  // Lua-accessible
 	bool DoWithEntityByID(UInt32 a_EntityID, cLambdaEntityCallback a_Callback, bool & a_CallbackResult);  // Lambda version
 
+	/** Calls the callback for each tyEntity; returns true if all block entities processed, false if the callback aborted by returning true
+	tBlocktype is a list of the blocktypes convertible to tyEntity which are to be called. If no block type is given the callback is called for every block entity
```

Comment:
`tBlockTypes are blocktypes convertible to tyEntity...`
(`list` could be mistaken for a `std::list`)